cfgov/unprocessed/css/organisms/email-popup.less

```diff
@@ -40,6 +40,10 @@
           top: unit( (@grid_gutter-width / 3) / @base-font-size-px, em);
           right: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
           cursor: pointer;
+
+          button {
+            border: none;
```

Comment:
Since this is overriding a value from an external library it wouldn't hurt to put a small comment here 

`// Override default border on Capital Framework button links.`

Jess may feel it's not necessary to override at all.